### ISSUE TITLE
Not enough arguments to `document.evaluate` in PaleMoon browser

### ISSUE THREAD

**Issue by silverwind** [bigskysoftware/htmx#2581]
[Here](https://github.com/bigskysoftware/htmx/blob/4eadc8f492bd90694455414d06f99801d950e94f/src/htmx.js#L1974-L1975), htmx calls [document.evaluate](https://developer.mozilla.org/en-US/docs/Web/API/Document/evaluate) with 4 arguments, but the function is specified on MDN to accept only 5 arguments:

````
evaluate(xpathExpression, contextNode, namespaceResolver, resultType, result)
````

The 5th argument (`result`) can be set to `null` but can not be left out and it appears the PaleMoon browser is strictly validating it while other browser accept `undefined` as well and treat it the same as `null`.

So, to fix this, I suggest passing `null` as the 5th argument.

**Comment by Telroshan** [comment]
Hey, good catch!
Feel free to submit a PR for [the v1 branch](https://github.com/bigskysoftware/htmx/tree/v1) as `dev` is now htmx 2, which doesn't use `document.evaluate` anymore but `XPathEvaluator.evaluate`, which [seems to accept from 2 to 5 parameters](https://developer.mozilla.org/en-US/docs/Web/API/XPathEvaluator/evaluate) - you might want to confirm that the latter behaves as expected in your browser though

**Comment by silverwind** [comment]
Closing as per https://github.com/bigskysoftware/htmx/pull/2592#issuecomment-2153730573. Will be fixed in upcoming PaleMoon 33.2.0.
